chia/wallet/wallet_node.py

```diff
@@ -1302,8 +1302,7 @@ async def validate_received_state_from_peer(
         fork_height: Optional[uint32],
     ) -> bool:
         """
-        Returns all state that is valid and included in the blockchain proved by the weight proof. If return_old_states
-        is False, only new states that are not in the coin_store are returned.
+        Returns all states that are valid and included in the blockchain proved by the weight proof.
```

Comment:
the function returns a `bool`, not "all states". It's not obvious that the updated version of the comment is correct